### ISSUE TITLE
Library Validation fails when trying to use vterm

### ISSUE THREAD

**Issue by mikeyobrien** [railwaycat/homebrew-emacsmacport#192]
When attempting to open a vterm, It fails with the following:

```
byte-code: Module could not be opened: "/Users/mikeyobrien/.emacs.d/.local/straight/build/vterm/vterm-module.so", "dlopen(/Users/mikeyobrien/.emacs.d/.local/straight/build/vterm/vterm-module.so, 1): no suitable image found.  Did find:
	/Users/mikeyobrien/.emacs.d/.local/straight/build/vterm/vterm-module.so: code signature in (/Users/mikeyobrien/.emacs.d/.local/straight/build/vterm/vterm-module.so) not valid for use in process using Library Validation: mapped file has no cdhash, completely unsigned? Code has to be at least ad-hoc signed."
```

This works as expected when using the default emacs package in homebrew,
`brew cask install emacs`

**Comment by railwaycat** [comment]
It looks like the app signature prevent the pre-built bundle  (from cask or download from release page) load modules. 

I may have to ship bundles without app signature on the upcoming cask build, but I'm afraid that will cause an annoying warning when they be opened first time.

**Comment by railwaycat** [comment]
fixed on [release 26.3-mac-7.9.1](https://github.com/railwaycat/homebrew-emacsmacport/releases/tag/emacs-26.3-mac-7.9.1)
